apps/api/src/services/db/transactionsService.ts

```diff
@@ -62,7 +63,9 @@ export async function getTransaction(hash: string) {
     ]
   });
 
-  if (!tx) return null;
+  if (!tx) {
+    return null;
```

Comment:
how come is this not failing a type check? `ApiTransactionResponse` can't be null

Reply:
not sure, but I'll modify the return type with `| null`